README.md

```diff
@@ -91,7 +91,10 @@ output {
    }            
 }               
 ```
+### ecs_compatibility
```

Comment:
Might be worthwhile adding a new section Support for ECS Compatibility otherwise it looks like it's a part of Configuration.